# =============== INITIAL JOINT STATE ===============
JOINT_1: 2.0690
JOINT_2: 0.9730
JOINT_3: 0.0430
JOINT_4: 0.7970
JOINT_5: -2.0340
JOINT_6: 1.6820
JOINT_7: 3.0010

# =============== ROBOT PARAMETERS ===============
GAIN: 3
SAMPLING_TIME: 0.01

# =============== MANIPULABILITY PARAMETERS (NOT USED) ===============
ENABLE_MANIPULABILITY: false
DERIVATIVE_STEP: 0.0001
NULL_GAIN: 5

# =============== PLANNER PARAMETERS ===============
TIME_TO_GRAB: 20
TIME_TO_RELEASE: 20
TIME_DELAY: 0.5
APPROACH_DISTANCE: 0.07
ZONE1_THRESHOLD: 0.3
ZONE2_THRESHOLD: 0.6